wfd/public/peer.h

```diff
@@ -44,8 +44,9 @@ class Peer {
     /**
      * The implementation should send the RTSP data over the network
      * @param data data to be send
+     * @return false if still sending previous message
```

Comment:
shouldn't it be handled by the client?
state machine won't handle 'false' appropriately